mesheryctl/pkg/utils/helpers.go

```diff
@@ -142,8 +138,9 @@ func installprereq() {
 	log.Info("Prerequisite Docker Compose is installed.")
 }
 
-func isMesheryRunning() bool {
-	op, err := exec.Command("docker-compose", "-f", dockerComposeFile, "ps").Output()
+// IsMesheryRunning checks if the meshery server containers are up and running
+func IsMesheryRunning() bool {
```

Comment:
Pascal case IsMesheryRunning

Reply:
These are common utility functions that are used in the internal package hence need to be exported using Pascal case. Camel case keeps the declaration package local

Reply:
Ah, sorry forgot the golang syntax

Reply:
Can you update the branch with master, failing tests are fixed on master ?

Reply:
Context switching from one language to the next is pain in the butt